content/sensu-go/5.18/release-notes.md

```diff
@@ -54,6 +55,22 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
 
 ---
 
+## 5.18.1 release notes
+
+**March 10, 2020** &mdash; The latest release of Sensu Go, version 5.18.1, is now available for download.
+
+This release fixes bugs that caused SQL migration failure on PostgreSQL 12, nil pointer panic due to OICD login, and sensu-backend restart upon agent disconnection. It also includes a reliability improvement &emdash; a change to use the gRPC client rather than the embedded etcd client.
```

Comment:
```suggestion
This release fixes bugs that caused SQL migration failure on PostgreSQL 12, nil pointer panic due to OICD login, and sensu-backend restart upon agent disconnection.
It also includes a reliability improvement &emdash; a change to use the gRPC client rather than the embedded etcd client.
```